contrib/devtools/update-translations.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python3
 # Copyright (c) 2014 Wladimir J. van der Laan
 # Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# file COPYING or http://www.opensource.org/licenses/mit-license.php
```

Comment:
Please do not make random changes to the license header here. Keep it the same as in every single other file.

Reply:
Sorry for not mentioning my reasoning - working in ubuntu text editor, the link was only displayed properly (italics) when the dot was removed. But I see now other headers all contain dots at the end.
Will add the dot back.